pkgs/applications/window-managers/lesbar/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  stdenv,
+  fetchFromSourcehut,
+  pkgs,
+}:
+stdenv.mkDerivation (finalAttrs: let
+  version = "1.1.0";
+in {
+  inherit version;
+  pname = "lesbar";
```

Comment:
I think you should use this instead:
```
stdenv.mkDerivation rec {
  pname = "lesbar";
  version = "1.10";
  ...
```
rec allows you to use attributes defined in the same scope

Reply:
apparently `finalAttrs` is now preferred

Reply:
Interesting...